src/cljs/athens/devcards/blocks.cljs

```diff
@@ -161,7 +161,7 @@
          [:> mui-icons/KeyboardArrowDown {:style {:font-size "16px"}}]]
         [:span (use-style block-disclosure-toggle-style)])
       [:a (use-style block-indicator-style {:class (if closed? "closed" "open") :on-click #(navigate-page uid)})]
-      [:span (parse-and-render string)]]
+      [:<> (parse-and-render string)]]
```

Comment:
I think here we should just write `[parse-and-render string]`

https://github.com/reagent-project/reagent/blob/master/doc/UsingSquareBracketsInsteadOfParens.md